src/styles/main.scss

```diff
@@ -2,6 +2,336 @@
 @import "fonts";
 @import "typography";
 
+@font-face {
+  font-family: Avenir; /* Имя шрифта */
+  src: url(../fonts/Avenir/Avenir-Book.ttf); /* Путь к файлу со шрифтом */
```

Comment:
do not leave comments like this in production code

Reply:
done